pkg/artifactory/resource_artifactory_replication_config_test.go

```diff
@@ -66,6 +66,10 @@ func testAccCheckReplicationDestroy(id string) func(*terraform.State) error {
 
 		replica, resp, err := client.V1.Artifacts.GetRepositoryReplicationConfig(context.Background(), rs.Primary.ID)
 
+		if resp == nil {
```

Comment:
Check `err` first. If you check error first you probably never need to check if `resp` is `nil`.